### PR TITLE
[DOCS-7249] Update Alfresco SDK version to 4.6

### PR DESCRIPTION
**Pull request by oliveralfresco2018**

### CODE REVIEW

File: content-services/latest/support/index.md [hunk 1]
```diff
@@ -85,6 +85,7 @@ Choose a combination of products to build your own Supported Stack. If anything
 | Alfresco Google Docs Integration 3.4 | |
 | Alfresco Content Services SDK 5.2 | |
 | Alfresco Content Services SDK 4.5 | |
+| Alfresco Content Services SDK 4.6 | |
```
Comment (anxumalo) [inline content-services/latest/support/index.md:88]: Moving the latest version to be first

```suggestion
| Alfresco Content Services SDK 4.6 | |
| Alfresco Content Services SDK 4.5 | |
```

**Review by anxumalo — COMMENTED** [review]
Adding initial comments .. more to come

File: content-services/latest/develop/sdk.md [hunk 2]
```diff
@@ -206,7 +203,7 @@ Alfresco recommends that you keep up-to-date with all the Docker releases. If yo
     14: 4.2.0
     15: 4.3.0
     15: 4.4.0
-    16: 4.5.0   
+    16: 4.6.0
```
Comment (anxumalo) [inline content-services/latest/develop/sdk.md:206]: Usually the previous version remains in this list - you can check it if you follow the "Getting started" steps.

File: content-services/latest/develop/sdk.md [hunk 3]
```diff
@@ -2560,7 +2511,7 @@ So, taking into account the previous section, let's see how the integration test
                         <artifactItem>
                             <groupId>org.apache.httpcomponents</groupId>
                             <artifactId>httpclient</artifactId>
-                            <version>4.5.2</version>
+                            <version>4.6.2</version>
```
Comment (anxumalo) [inline content-services/latest/develop/sdk.md:2514]: I'm not sure if this version number is related to the SDK version - please check

File: content-services/latest/develop/oop-sdk.md
[the remark this answers is not in the document]
Comment (oliveralfresco2018) [reply content-services/latest/develop/oop-sdk.md:1]: Has been done in https://github.com/Alfresco/docs-alfresco/pull/1074

File: content-services/latest/develop/sdk.md
[the remark this answers is not in the document]
Comment (oliveralfresco2018) [reply content-services/latest/develop/sdk.md:1]: Has been done in https://github.com/Alfresco/docs-alfresco/pull/1074

File: content-services/latest/develop/sdk.md (hunk 3 above)
Comment (oliveralfresco2018) [reply content-services/latest/develop/sdk.md:2514]: Has been done in https://github.com/Alfresco/docs-alfresco/pull/1074

File: content-services/latest/develop/sdk.md (hunk 2 above)
Comment (oliveralfresco2018) [reply content-services/latest/develop/sdk.md:206]: Has been done in https://github.com/Alfresco/docs-alfresco/pull/1074

**Comment by oliveralfresco2018** [comment]
Closing this PR, work has been done in https://github.com/Alfresco/docs-alfresco/pull/1074

**Pull request closed without merging**